apps/web/src/pages/api/feedback.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-case-declarations */
+import type { NextApiRequest, NextApiResponse } from "next";
+
+const webhooks = [
+  process.env.FEEDBACK_WEBHOOK_URL ??
+    "https://discord.com/api/webhooks/abc/123",
```

Comment:
This is not a valid default so I'd rather remove it and make the variable optional